eth2near/eth2near-block-relay-rs/src/eth2near_relay.rs

```diff
@@ -275,6 +280,8 @@ impl Eth2NearRelay {
 
         let signature_slot =
             last_finalized_slot_on_near + self.current_gap_between_finalized_and_signature_slot;
+        debug!(target: "relay", "Chosen signature slot {}", signature_slot);
```

Comment:
This is should be a `trace` level message
```suggestion
        trace!(target: "relay", "Chosen signature slot {}", signature_slot);
```

Reply:
fixed: https://github.com/aurora-is-near/rainbow-bridge/pull/775/commits/e7455a096a6599199ee548c186f66717effc1963